releases/release-1.19/release_team.md

```diff
@@ -0,0 +1,16 @@
+# Kubernetes 1.19 Release Team
+
+| **Role**         | **Name** (**GitHub / Slack ID**)                                                                                                                        | **Shadow Name(s) (GitHub / Slack ID)**                                                                                              |
+| ---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- |
+| Lead             | Taylor Dolezal ([@onlydole](https://github.com/onlydole) / Slack: [@onlydole](slack://user?team=T09NY5SBT&id=D3KH4F413))                                | Bob Killen ([@mrbobbytables](https://github.com/mrbobbytables) / Slack: [@mrbobbytables](slack://user?team=T09NY5SBT&id=DMK8USDT5)) |
```

Comment:
Remove the Slack links from each user. They don't render on every browser, so they're effectively invisible unless you inspect the raw document.